collectors/node.d.plugin/bigbluebutton/bigbluebutton.node.js

```diff
@@ -0,0 +1,196 @@
+// this is the mymodule definition
+var crypto = require("crypto");
+var https = require("https");
+var netdata = require("netdata");
+var url = require("url");
+
+var bigbluebutton = {
+  charts: {},
+
+  base_priority: 60000,
+
+  processResponse: function(service, data) {
+    /* send information to the Netdata server here */
+    if (data !== null) {
+      if (service.added !== true) {
+        service.commit();
+      }
+
+      var conferences = (data.match(/<meetingID>/g) || []).length;
+
+      var id = "bbb_" + service.name + ".conferences";
+      var chart = bigbluebutton.charts[id];
+
+      if (typeof chart === "undefined") {
+        chart = {
+          id: "bbb_conferences", // the unique id of the chart
+          name: "BBB conferences", // the unique name of the chart
+          title: "BBB Current conferences", // the title of the chart
+          units: "conferences", // the units of the chart dimensions
+          family: "conferences", // the family of the chart
+          context: "bigbluebutton.conferences", // the context of the chart
+          type: netdata.chartTypes.line, // the type of the chart
+          priority: bigbluebutton.base_priority + 1, // the priority relative to others in the same family
+          update_every: service.update_every, // the expected update frequency of the chart
+          dimensions: {
+            conferences: {
+              id: "conferences", // the unique id of the dimension
+              name: "", // the name of the dimension
+              algorithm: netdata.chartAlgorithms.absolute, // the id of the netdata algorithm
+              multiplier: 1, // the multiplier
+              divisor: 1, // the divisor
+              hidden: false // is hidden (boolean)
+            }
+          }
+        };
+        chart = service.chart(id, chart);
+        bigbluebutton.charts[id] = chart;
+      }
+
+      service.begin(chart);
+      service.set("conferences", conferences);
+      service.end();
+
+      var sumXmlAttr = function(name) {
+        var sum = 0;
+        var r = new RegExp("<" + name + ">([0-9]+)</" + name + ">", "g");
+        var match = r.exec(data);
+        while (match !== null) {
+          sum += parseInt(match[1]);
```

Comment:
```suggestion
          sum += parseInt(match[1], 10);
```

Pointed out by Codacy. Without the radix being explicitly specified there's a risk of this not actually decoding the integer as base 10.

Reply:
@Ferroin Thanks for your review, I addressed your feedbacks.